analysis/priestdiscipline/src/CHANGELOG.tsx

```diff
@@ -5,6 +5,7 @@ import { SpellLink } from 'interface';
 
 
 export default [
+  change(date(2022,25,6), <>General Draven HPS increases added</>, Hana),
```

Comment:
Date format should be year/month/day - apparently it just adds up all the months and renders this as being in 2024 LOL 
![image](https://user-images.githubusercontent.com/7143486/177555071-2601f029-1412-4ae8-8f62-437efeb87832.png)